src/components/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import '../styles/nav.css';
+
+const Nav = () => (
+  <nav>
+    <h1>Bookstore CMS</h1>
+    <ul className={Nav.ul}>
+      <li id="nav-book"><Link to="/">BOOKS</Link></li>
+      <li id="nav-categories"><Link to="/categories">CATEGORIES</Link></li>
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct font colour` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the attached image and table.

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/67fdda37-30a1-4e0d-aeae-d9f33ea05897)

</td>
<td>

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/66a5e3f2-3822-41f7-b51b-39e9bd04afa8)

</td>
</tr>
</table>